XamarinCommunityToolkit/RegularExpressions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace XamarinCommunityToolkit
+{
+    public static class RegularExpressions
+    {
+        internal static string Email => @"^(?("")("".+?(?<!\\)""@)|(([0-9a-z]((\.(?!\.))|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w])*)(?<=[0-9a-z])@))(?(\[)(\[(\d{1,3}\.){3}\d{1,3}\])|(([0-9a-z][-0-9a-z]*[0-9a-z]*\.)+[a-z0-9][\-a-z0-9]{0,22}[a-z0-9]))$";
+        internal static string Password => @"^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$";
+        internal static string Phone => @"/^\b\d{3}[-.]?\d{3}[-.]?\d{4}\b$/";
```

Comment:
Phone numbers are formed differently in each country and I don't think there's something that is universally acknowledged across the world as a valid phone number. Typically I would assume users want to validate phone numbers from specific countries, so I don't see this one getting used a lot.